docs/user/ovirt/customization.md

```diff
@@ -0,0 +1,55 @@
+# oVirt IPI customization
+
+Before you begin see the [generic cluster customization doc](docs/user/customization.md) 
+
+Installation on oVirt allows various customization of the created resources. 
+Those changes are made prior to the cluster install as they will affect the  
+creation of the resources, and we want to make sure all of the resources have 
+the same set of definitions.
+ 
+The installer-created resources are:
+- RHCOS qcow image (uploaded on cluster creation) 
+- Template from the pinned RHCOS qcow disk
+- master Vms, and disks, from the template.
+
+A dedicated VLAN network per cluster may come in the future. 
+
+Customization can be made in prior to the installation or after the cluster is created.
+ 
+## Customize MEM/CPU
+
+Env variables for the install time:
+
+| Env variable  | Description |
+| ---  | --- |
+| TF_VAR_ovirt_template_disk_size_gib | The size of the template disk for worker/nodes in GiB | 
+| TF_VAR_ovirt_master_mem | Master VM memory in MiB  | 
+| TF_VAR_ovirt_master_cpu | Master number of CPUs | 
+| TF_VAR_ovirt_template_mem | Template VM memory in MiB | 
+| TF_VAR_ovirt_template_cpu | Template number of CPUs |
+
+Both masters and workers are created from the same template, but since master 
+have different resources usage pattern it is overriden. Its important to node 
+that every future worker will be created from the template.
+
+## Template editing
+
+Its possible to use a specific template and not upload the pinned one. The right 
+way to it is to let the installer upload the pinned disk image, create the template 
+and then stop the installation for editing.
+A created template would have the name of the [`infraID`](pkg/asset/installconfig/clusterid.go) 
+Now this template can be edited, for example to contain 2 nics, and then its name
+can be passed as an env variable - covered (here as well)[docs/dev/]  
+
+| Env variable  | Description |
+| ---  | --- |
+| OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE | oVirt's template name or a URL of a qcow image to load |
```

Comment:
Same, user facing options should be available in install-config, ex: https://github.com/openshift/installer/pull/3170